packages/ui/src/forms/ImagePicker/index.tsx

```diff
@@ -9,7 +12,12 @@ interface ContainerProps {
 }
 
 interface Props {
```

Comment:
for the props interfaces with use the following convention:
_Component Name + "Props"_
**Example** ImagePickerProps